app/views/GermlineView/components/Report/index.tsx

```diff
@@ -98,8 +103,10 @@ const GermlineReport = ({
     />
   ), [onEdit]);
 
-  const handleConfirmDelete = useCallback(async (confirm) => {
+  const handleClose = useCallback(async (confirm) => {
```

Comment:
non-blocking: given there's a couple popups in this component, I might make this name more verbose so it's clearer without looking where it's called which popup it's for. maybe `handleCloseDeleteAlert` or similar